steps/nitro-android/4.5.0/step.yml

```diff
@@ -0,0 +1,285 @@
+source:
+  git: https://github.com/nitro-build/bitrise-step-nitro-android.git
+  commit: 74c8e034270873ea9849db1de6bdcb8c146220d7
+host_os_tags:
+- ubuntu-16.04
+project_type_tags:
+- android
+- react-native
+type_tags:
+- build
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: wget
+  - name: pigz
+  - name: gnu-sed
+  - name: jq
+  apt_get:
+  - name: pigz
+  - name: file
+  - name: jq
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    is_expand: true
+    summary: The directory within your project, in which your code is located. Leave
+      this field empty if your code is not located in a subdirectory
+    title: Root directory
+  root_directory: ./
+- flavor: null
+  opts:
+    is_expand: true
+    summary: Set product flavor for Gradle build variant
+    title: Flavor
+- opts:
+    category: App Versioning
+    is_expand: true
+    summary: The version name for the app
+    title: Version name
+  version_name: null
+- opts:
+    category: App Versioning
+    is_expand: true
+    summary: The version code for the app
+    title: Version code
+  version_code: null
+- disable_version_name_from_package_json: "no"
+  opts:
+    category: App Versioning
+    is_expand: true
+    summary: By default will get the 'version' field from package.json and set the
+      version name
+    title: Disable automatic version name configuration
+    value_options:
+    - "yes"
+    - "no"
+- disable_version_code_auto_generation: "no"
+  opts:
+    category: App Versioning
+    is_expand: true
+    summary: By default will generate a timestamp based number and set the version
+      code
+    title: Disable automatic version code generation
+    value_options:
+    - "yes"
+    - "no"
+- keystore_url: $BITRISEIO_ANDROID_KEYSTORE_URL
+  opts:
+    category: App Signing
+    is_expand: true
+    summary: Keystore url
+    title: Keystore url
+- keystore_password: $BITRISEIO_ANDROID_KEYSTORE_PASSWORD
+  opts:
+    category: App Signing
+    is_expand: true
+    is_sensitive: true
+    summary: Keystore password
+    title: Keystore password
+- keystore_key_alias: $BITRISEIO_ANDROID_KEYSTORE_ALIAS
+  opts:
+    category: App Signing
+    is_expand: true
+    summary: Keystore alias
+    title: Keystore alias
+- keystore_key_password: $BITRISEIO_ANDROID_KEYSTORE_PRIVATE_KEY_PASSWORD
+  opts:
+    category: App Signing
+    is_expand: true
+    is_sensitive: true
+    summary: Keystore key password
+    title: Keystore key password
+- cache_provider: s3
+  opts:
+    category: Caching
+    description: |-
+      Choose the provider where cache artifacts will be persisted:
+      - `fs`: File system
+      - `s3`: Amazon - Simple Storage Service
+    is_expand: true
+    summary: Choose the provider where cache artifacts will be persisted
+    title: Cache provider
+    value_options:
+    - fs
+    - s3
+- disable_cache: $NITRO_DISABLE_CACHE
```

Comment:
Environment Variables shouldn't be used as default values, which are not exposed by Bitrise or as output by another Step. Read more here: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#step-input-keys-and-values

Reply:
Hi @godrei thanks for the feedback! will take into account on further PRs